server/controllers/profile.js

```diff
@@ -0,0 +1,73 @@
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /users
+// @desc Search for users
+// @access Private
+
+exports.createUser = asyncHandler(async (req, res, next) => {
+  // Extracting fields from body
+  let firstName = req.body.firstName;
+  let lastName = req.body.lastName;
+  let description = req.body.description;
+  let availability = req.body.availability;
+  const profile = await Profile.create({
+    firstName: firstName,
+    lastName: lastName,
+    description: description,
+    availability: availability,
+  });
+});
+
+exports.findProfile = asyncHandler(async (req, res, next) => {
+  // extracting id from url
+  let Id = req.params.id;
```

Comment:
Avoid 
```suggestion
  const id = req.params.id;
```

Avoid using PascaleCase for variables and because this variable doesn't change, it should be a constant